invenio_app_ils/cli.py

```diff
@@ -263,7 +263,7 @@ def persist(self):
 class DocumentGenerator(Generator):
     """Document Generator."""
 
-    DOCUMENT_TYPES = ["BOOK", "STANDARD", "PROCEEDING", "JOURNAL"]
+    DOCUMENT_TYPES = ["BOOK", "STANDARD", "PROCEEDING"]
```

Comment:
it is `PERIODICAL_ISSUE`, just to avoid any misunderstanding, right?

Reply:
I wanted to avoid generating random periodical issues that didn't belong to a serial. Maybe we don't even need these constants, we don't use them elsewhere.